src/cat.sol

```diff
@@ -96,30 +99,30 @@ contract Cat is LibNote {
     // --- Administration ---
     function file(bytes32 what, address data) external note auth {
         if (what == "vow") vow = VowLike(data);
-        else revert();
+        else revert("Cat/file-wrong-param");
```

Comment:
nit: "unrecognized" instead of "wrong" is a little more descriptive, but not a big deal if you're trying to prioritize brevity

Reply:
Good, I'll change it.